### PR TITLE
Rename 'UpgradeOverlay' because of the new version of OpenRA

### PR DESCRIPTION
**Pull request by cookgreen**

### CODE REVIEW

File: rules/defaults.yaml [hunk 1]
```diff
@@ -83,8 +83,6 @@
 		Conditions: crate-firepower, crate-damage, crate-speed
 
 ^IronCurtainable:
-	UpgradeOverlay@ironcurtain:
```
Comment (m3ntalsp00n) [inline rules/defaults.yaml]: Should be renamed to WithColoredOverlay not removed entirely.

Comment (cookgreen) [inline rules/defaults.yaml]: okay

Comment (cookgreen) [inline rules/defaults.yaml]: changed

**Comment by cookgreen** [comment]
Fixed

**Pull request closed without merging**